### PR TITLE
APC.PowerChutePersonalEdition version 3.1.0

### PR DESCRIPTION
**Pull request by mddtest**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19373)

### CODE REVIEW

**Comment by mddtest** [comment]
I will get that fixed tomorrow. The sandbox tool is broken so installed on main pc no issue

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

File: manifests/a/APC/PowerChutePersonalEdition/3.1.0/APC.PowerChutePersonalEdition.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: APC.PowerChutePersonalEdition
+PackageVersion: 3.1.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://download.schneider-electric.com/files?p_File_Name=PCPE_3.1.0.exe
+  InstallerSha256: 06CDB01B2E23ED72452FF090212BA72B5A250C16C41485ECD05D4AC9F75E25FF
+ManifestType: installer
```
Comment (mddtest) [inline manifests/a/APC/PowerChutePersonalEdition/3.1.0/APC.PowerChutePersonalEdition.installer.yaml:11]: InstallerSwitches:
    Silent: /silent
    SilentWithProgress: /silent

Comment (mddtest) [inline manifests/a/APC/PowerChutePersonalEdition/3.1.0/APC.PowerChutePersonalEdition.installer.yaml:11]: InstallerSwitches:
    Silent: /silent
    SilentWithProgress: /silent

**Comment by mddtest** [comment]
> 
> 
> image

This will come up because no APC unit is found.  so not sure if we will be able to add this one as the bot wont have a APC.

File: manifests/a/APC/PowerChutePersonalEdition/3.1.0/APC.PowerChutePersonalEdition.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/a/APC/PowerChutePersonalEdition/3.1.0/APC.PowerChutePersonalEdition.installer.yaml:11]: Switches are already included in the manifest as part of `InstallerType: nullsoft` which runs the manifest with `/S` from https://github.com/microsoft/winget-cli/blob/cf6f8e58195bf033a1a5872f7b800866cd27a586/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L362-L368

`/silent` is not a valid switch for nullsoft applications.

**Comment by denelon** [comment]
@mddtest thanks for letting us know about the hardware requirement. Now we need a giant UPS to go with that awesome gaming rig we need to validate some of the NVidia software 😁

**Comment by mddtest** [comment]
I can volunteer if given the funds still rocking a 2600k.,🤣

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**